docs/xks/developer-guide/security.md

```diff
@@ -51,6 +51,29 @@ For example:
           runAsUser: 1000
 ```
 
- The `runAsUser` field specifies that for any Containers in the Pod, all processes run with user ID 1000. The runAsGroup field specifies the primary group ID of 1000 for all processes within any containers of the Pod
+ The `runAsUser` field specifies that for any Containers in the Pod, all processes run with user ID 1000. The runAsGroup field specifies the primary group ID of 1000 for all processes within any containers of the Pod. If this field is not set, the primary group ID of the containers will be root. Using `runAsNonRoot: true` makes it impossible for containers to run as root, it will require a defined non-zero numeric USER directive defined in the container image.
 
-You can read more in the [official documentation](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/)
+You can read more in the official documentation on [Security Context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/)
+ and [Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/)
+
+### EmptyDir
+
+An emptyDir volume is created when a Pod is assigned to a node and exists while that Pod is running. The emptyDir volume is initially empty. All containers in the Pod can read and write the same files in the emptyDir volume and can be mounted to different paths in each container. When the Pod is removed, the data in the emptyDir is deleted permanently.
+An example configuration of a basic emptyDir is:
+
+```yaml
+  volumes:
+  - name: temp
+    emptyDir: {}
+```
```

Comment:
We don't need to explain what emptyDir does in this case. But rather why do we need emptyDir when using securityContext and our default OPA settings?
In short we don't allow data to be written on our root file system. So if you have written a application that have to be able to do this you need to mount a disk, most of the time a emptyDir to be able to write some small amount of data.
For example .net have to write data to /tmp or else it won't start as intended.